data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -506,50 +401,32 @@ private void handleTokTab(ASTNode nextToDestinationNode) {
 			node = (ASTNode) (nextToDestinationNode.getChild(0));
 		else
 			node  = nextToDestinationNode;
-//		String table_name = (nextToDestinationNode.getChildCount() == 1) ?
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) :
-//				getUnescapedName((ASTNode) nextToDestinationNode.getChild(0)) +
-//						"." + getUnescapedName((ASTNode) nextToDestinationNode.getChild(1));
 		String table_name = (node.getChildCount() == 1) ?
 				getUnescapedName((ASTNode) node.getChild(0)) : getUnescapedName((ASTNode) node.getChild(1));
 		outputTableList.add(table_name);
-		System.out.println("Output Table name in handleTokTab = " + table_name);
+		LOGGER.info("Output Table name in handleTokTab = " + table_name);
 		Table table = new Table(table_name, defaultHiveDbName, null, EntityType.OUTTYPE);
 		outTableNodes.add(table);
 	}
 
 	private void handleTokDir(ASTNode nextToDestinationNode, ASTNode queryNode) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "nextToDestinationNode". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)